lib/ruby_lsp/requests/support/rubocop_diagnostic.rb

```diff
@@ -19,6 +19,12 @@ class RuboCopDiagnostic
           T::Hash[Symbol, Integer],
         )
 
+        # Cache cops to attach URLs to diagnostics. Only built-in cops for now.
+        COP_TO_DOC_URL = T.let(
+          RuboCop::Cop::Registry.global.to_h,
```

Comment:
note that invoking to the RuboCop global registry at the top-level is sensitive to load order.

Reply:
I think it should be fine. We require `rubocop` before this class is loaded.

Reply:
Yep, it's also best if this file isn't required until after RuboCop has been _run_ for the first time (which is the case right now). This is because the first run is when non-built-in cops get added to the global registry.

E.g. in the ruby lsp codebase `RuboCop::Cop::Registry.global.length` is 544 before this line and 635 after it: 

https://github.com/Shopify/ruby-lsp/blob/5b732ebe1d6c4dbaf40a4852946fd1de89830590/lib/ruby_lsp/requests/support/rubocop_runner.rb#L90

Doc links for non-built-in cops don't even work right now, so it's sort of moot, but seems worth being aware of!